PusherServer/TriggerResult.cs

```diff
@@ -1,14 +1,49 @@
-﻿using System;
-using System.Net;
+﻿using PusherServer.Exceptions;
+using PusherServer.Util;
 using RestSharp;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Web.Script.Serialization;
 
 namespace PusherServer
 {
     internal class TriggerResult: RequestResult, ITriggerResult
     {
+        private readonly IDictionary<string, string> _eventIds;
+
         public TriggerResult(IRestResponse response):
             base(response)
         {
+            EventIdData eventIdData = null;
+            try
+            {
+                JavaScriptSerializer serializer = new JavaScriptSerializer();
+                eventIdData = serializer.Deserialize<EventIdData>(response.Content);
+            }
+            catch (Exception)
+            {
+                // Allow this to be swallowed. It will be caught in the following (eventIdData == null) check.
+            }
+
+            if (eventIdData == null || eventIdData.event_ids == null || eventIdData.event_ids.Count == 0)
```

Comment:
:+1: this was the change that resulted in me asking @j-siddle to check backwards compatibility requirements.